sql/share/errmsg-utf8.txt

```diff
@@ -10780,3 +10780,5 @@ ER_CM_OPTION_MISSING_REQUIREMENT
         eng "CHANGE MASTER TO option '%s=%s' is missing requirement %s"
 ER_SLAVE_STATEMENT_TIMEOUT 70100
         eng "Slave log event execution was interrupted (slave_max_statement_time exceeded)"
+ER_SECURE_TRANSPORT_REQUIRED 08004
```

Comment:
let's try to have it the same number as in MySQL, if possible

Reply:
- MySQL 5.7: https://github.com/mysql/mysql-server/blob/5.7/sql/share/errmsg-utf8.txt#L7574-L7575
- MySQL 8.0: https://github.com/mysql/mysql-server/blob/8.0/share/messages_to_clients.txt#L7650-L7651

Reply:
How did you do it? I don't see how it was done :(

Reply:
Isn't it defined only through `sql/share/errmsg-utf8.txt` ? If so, I modified the error number to match the new number